src/services/dbtLineageService.ts

```diff
@@ -260,7 +260,7 @@ export class DbtLineageService {
     return g.get(key)?.nodes.length || 0;
   }
 
-  private cancellationTokenSource: CancellationTokenSource | undefined;
+  cancellationTokenSource: CancellationTokenSource | undefined;
```

Comment:
Create method and make private or move this to doc edit panel. I think it is weird for the service to have this state.